configure.sh

```diff
@@ -140,6 +126,24 @@ export HTML_DIRECTORY=$ROOT_WEB_DIRECTORY
 mkdir /var/www/html/competitions
 chown vagrant:www-data /var/www/html/competitions
 
+# Install SimpleBook
+echo "INSTALL SIMPLEBOOK SERVER"
+cd /home/vagrant/torque-sites/base/simplebook/ansible
+envsubst < inv/local/group_vars/all.tmpl > inv/local/group_vars/all
+ansible-playbook simplebook.yml -i inv/local
+
+# Overwrite SimpleBook installation with mounted folder
```

Comment:
If my subsequent comment (see immediately below) is correct, could we update this comment to be a little more specific? That will help *future Will* understand what is going on.